examples/torch/sac_half_cheetah.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+"""
+This is an example to train a task with DDPG algorithm written in PyTorch.
+
+Here it creates a gym environment InvertedDoublePendulum. And uses a DDPG with
+1M steps.
+
+"""
+import gym
+import torch
+from torch.nn import functional as F  # NOQA
+from torch import nn as nn
+
+from garage.envs import normalize
+from garage.envs.base import GarageEnv
+from garage.experiment import LocalRunner, run_experiment
+from garage.replay_buffer import SimpleReplayBuffer
+from garage.torch.algos import SAC
+from garage.torch.policies import TanhGaussianMLPPolicy
+from garage.torch.q_functions import ContinuousMLPQFunction
+
+
+def run_task(snapshot_config, *_):
+    """Set up environment and algorithm and run the task."""
+    runner = LocalRunner(snapshot_config)
+    env = GarageEnv(normalize(gym.make('HalfCheetah-v2')))
+
+    policy = TanhGaussianMLPPolicy(env_spec=env.spec,
+                               hidden_sizes=[256, 256],
+                               hidden_nonlinearity=nn.ReLU,
+                               output_nonlinearity=None)
+
+    qf1 = ContinuousMLPQFunction(env_spec=env.spec,
```

Comment:
if `qf1`/`qf2`, by definition in the algorithm, need to be the same network, then i suggest they be copied by the algo.

Reply:
I'm not convinced they need to be? Also, they definitely need to be initialized differently, which is tricky to do without assuming an initialization strategy in the algorithm or extending the `torch.nn.Module` API.